src/routes/blog/ide-ux-bof-scipy-2024/+page.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "Scientific IDE UX Birds of a Feather session at SciPy 2024"
+author: camgerlach
+pub_date: 2024-09-05
+category: News
+tags: community, outreach, media, team
+summary: The Spyder team hosted another Birds of a Feather session at SciPy 2024, this time on the topic of users' experiences (good and bad) with the UI/UX of scientific interfaces and IDEs, and how their developers can better serve users. Here, we share what we learned from the session, as well as a link to the full detailed community notes.
+---
+
```

Comment:
Perhaps we should add here the picture someone at NF took for the BoF.

Reply:
No worries. The photo is [here](https://x.com/Spyder_IDE/status/1813011498224263378) (I just took it from another NF post on X).

Reply:
Thanks; added

Reply:
Hmm, seems like its not displaying, even though AFAIK I've added the image in the same way as the other blog posts. Is there anything you can spot that I'm doing wrong?

Reply:
Hmm, looks like its working now. But its displaying the alt text as a caption, which is not correct.

Reply:
This has been solved by [PR 14](https://github.com/spyder-ide/spyder-website/pull/14). I will mark the conversation as resolved.